### ISSUE TITLE
'--with-default-names' in brew install grep not available anymore , also base16_onedark not working ?

### ISSUE THREAD

**Issue by souhaiebtar** [nicknisi/dotfiles#110]
Hello,
first thanks for your theme, just to know `brew install grep --with-default-names` not available anymore,
to get the same behavior, you use
```shell
brew install grep
export PATH="/usr/local/opt/grep/libexec/gnubin:$PATH"
```

about the `xterm-256color-italic`,
i had to manually enter

```shell
tic -x xterm-256color-italic.terminfo
```
to make it work

alos base16_onedark does not show as in your screenshot ( i installed base16-shell),

![onedarkNotworking](https://i.imgur.com/6i82Jwo.png)

have i missed anything, do you have any idea how to fix it ?

**Comment by nicknisi** [comment]
Thanks for the feedback! I addressed the grep install issue in aa113b7.

About the terminfo issue, I don't currently have that as part of the install, so running that was the proper step. 👍 

Finally, the screenshot in the picture is a bit dated from the current setup (sorry, I tend to tweak a lot). The main reason I'm using base16 is so that I can switch easily to another theme when/if I get bored by using the `base16_` shell commands. Here is a current screenshot of what the setup looks like.

<img width="1122" alt="image" src="https://user-images.githubusercontent.com/293805/59983698-5efb7580-95e8-11e9-9bbd-d087fb13333f.png">

Your screenshot is what I would expect base16-onedark to look like.

I'm still continuing to improve the initial installation process, and overall usability of what's in these dotfiles. To that end, I'll definitely incorporate this feedback into future improvements. Thanks!

**Comment by souhaiebtar** [comment]
@nicknisi, the thanks really goes to you
